README.md

```diff
@@ -43,7 +51,7 @@ cp -R "pesdk-html5-build-$VERSION/assets/"* vendor/assets/images
 
 4. Create new home controller with index page
 
-```bash 
+```ruby 
```

Comment:
This was actually right, as the command is executed using bash (in the shell) and not written in Ruby. But should not really effect the syntax highlighting anyway.

Reply:
So should I remove all syntax highlighting extensions?

Reply:
No, just change it back to bash :)